examples/canvas/src/lib.rs

```diff
@@ -1,10 +1,7 @@
-extern crate js_sys;
-extern crate wasm_bindgen;
-extern crate web_sys;
-
 use std::f64;
 use wasm_bindgen::prelude::*;
 use wasm_bindgen::JsCast;
+use web_sys;
```

Comment:
I think you can actually remove this `web_sys` line in 2018 too!

Reply:
Fixed! 🚀